content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1154 @@
+---
+slug: "stats19"
+title: "stats19: a package for road safety research"
+package_version: 0.2.0
+authors:
+  - Layik Hama
+  - Robin Lovelace
+date: 2019-02-18
+categories: blog
+topicid:
+tags:
+- R
+- software
+- packages
+- stats19
+- road safety 
+- open data UK
+---
+
+## Introduction
+
+**stats19** is a new R package that enables access to and processing of
+Great Britain’s official road traffic casualty database,
+[STATS19](https://data.gov.uk/dataset/cb7ae6f0-4be6-4935-9277-47e5ce24a11f/road-safety-data).
+It has been peer reviewed thanks to rOpenSci and is now published in the
+Journal of Open Source Software
+([JOSS](http://joss.theoj.org/papers/10.21105/joss.01181)) (Lovelace et
+al. 2019). For installation and the code, see its home in rOpenSci:
+<https://github.com/ropensci/stats19>, and the package documentation at
+<https://itsleeds.github.io/stats19/>.
```

Comment:
```suggestion
<https://https://ropensci.github.io/stats19/>.
```
please use the rOpenSci link here to be consistent with the package and post

Reply:
I'm not sure about this. @sckott says we can update the 'canonical' location of the package website. Trying to encourage Leeds uni to support open source and if it's prominently form ITS, it could help build the case for more open source projects developed in the institution and in the wider university culture. See where I'm coming from?

Reply:
Happy to listen to @sckott on this and I definitely see where you're coming from!